### ISSUE TITLE
Type and Year filters are not in sync

### ISSUE THREAD

**Issue by gauravpansari1991** [Sitecore/XM-Cloud-Introduction#427]
**Describe the bug**

**To Reproduce**
1. Go to https://mvp.sitecore.com/en/Directory
2. Click on **Commerce** Type Filter
3. It shows filtered Year facets, select **2024** from the year filter
4. It is showing 8 results for Technology and Ambassador MVPs
5. Also it should not show other **Type** filters like Ambassador and Technology

**Expected behavior**
2024 doesn't have any commerce MVPs, so either we should hide 2024 or make it disable with zero count

**Screenshots**
<img width="386" alt="image" src="https://github.com/Sitecore/XM-Cloud-Introduction/assets/12952868/37ae25d0-bfec-4ccf-b138-99f6935eef9d">

**Comment by sc-ivanlieckens** [comment]
@gauravpansari1991 this result is actually again expected. The 8 people you're looking at were Commerce MVPs in the past and are MVP in 2024. So the combination of the facets isn't happening exactly as you expect (it evaluates people and all their titles, not titles separately)

@mrtamasvarga we discussed this shortly "what feels logical" how facets behave inside themselves and towards other facets. Do you believe this should change?

**Comment by gauravpansari1991** [comment]
@sc-ivanlieckens  Makes sense, just for suggestion in this case we can show the Commerce Category as well in the profile while the facet is on.

**Comment by sc-ivanlieckens** [comment]
@gauravpansari1991 I'm not sure I understand entirely what you mean by that? The profile always shows all titles and is a different page? Are you referring to the result cards?

**Comment by gauravpansari1991** [comment]
@sc-ivanlieckens Yeah I am referring to profile cards, sorry for the confusion. Just a suggestion that we can also show the commerce category in profile cards.

**Comment by robearlam** [comment]
@mrtamasvarga - How do you want to handle this, do you want to leave the functionality as is, or do you want to update the Profile cards to list all categories that an MVP has been awarded in over the years as requested above?
